client/v2/autocli/util.go

```diff
@@ -11,12 +11,19 @@ import (
 
 // findSubCommand finds a sub-command of the provided command whose Use
 // string is or begins with the provided subCmdName.
+// It verifies the command's aliases as well.
 func findSubCommand(cmd *cobra.Command, subCmdName string) *cobra.Command {
 	for _, subCmd := range cmd.Commands() {
 		use := subCmd.Use
 		if use == subCmdName || strings.HasPrefix(use, subCmdName+" ") {
 			return subCmd
 		}
+
+		for _, alias := range subCmd.Aliases {
```

Comment:
As discussed yesterday in our sync @atheeshp.
This way, we can prevent a command to be created by having an alias as well instead of having to use skip.